api/krusty/fnplugin_test.go

```diff
@@ -630,3 +631,186 @@ metadata:
   name: env
 `)
 }
+
+func TestFnContainerMounts(t *testing.T) {
+	skipIfNoDocker(t)
+
+	th := kusttest_test.MakeHarness(t)
+	o := th.MakeOptionsPluginsEnabled()
+	fSys := filesys.MakeFsOnDisk()
+	b := MakeKustomizer(&o)
+	tmpDir, err := filesys.NewTmpConfirmedDir()
+	assert.NoError(t, err)
+
+	path, err := os.Getwd()
+	assert.NoError(t, err)
+
+	src := filepath.Join(path, "testdata", "charts") + string(filepath.Separator)
+	dst := filepath.Join(tmpDir.String(), "charts") + string(filepath.Separator)
+	assert.NoError(t, os.Mkdir(dst, 0777))
+
+	cmd := exec.Command("cp", "-r", src, dst)
```

Comment:
This test passes locally, but not in the Linux CI job. I'll dig into this again tomorrow.